packages/loader/container-loader/src/utils.ts

```diff
@@ -74,7 +74,7 @@ function convertSummaryToSnapshotWithEmbeddedBlobContents(
                 const blobId = uuid();
                 treeNode.blobs[key] = blobId;
                 const contentBuffer = typeof summaryObject.content === "string" ?
-                    stringToBuffer(summaryObject.content, "utf8") : summaryObject.content;
+                    stringToBuffer(summaryObject.content, "utf8") : summaryObject.content.buffer;
```

Comment:
This is not fully correct, as Uint8Array might be a partial view into its underlying array buffer.
You recently did slicing in ODSP driver for similar flow. We should create a helper function that does the right stuff (usage of full buffer or slice depending on situation).
Better fix I believe would be to move use one type, across all subsystems. I think Uint8Array is better, not sure why we chose ArrayBuffer in the first place

Reply:
Made the changes. I will open an issue to make changes from ArrayBuffer to Uint8 at other places.